### DISCUSSION TITLE
dspira-lessons/forum/general/old/

### DISCUSSION THREAD

**Discussion by PranavSanghavi** [WVURAIL/dspira-lessons#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nday91** [comment]
Hello.  Thank you for publishing the instructions for a Hydrogen line Feedhorn.  I have built one using the 2019 instructions and successfully detected a "bump" in the spectrum at 1.420GHz when I point the antenna at the Milky Way.

I am now trying to fine tune the antenna and I have a question.  The current instructions call for the probe to be placed 5.25cm from the end of the can but older DSPIRA instructions use a distance of 6.5cm and http://physicsopenlab.org/wp-content/uploads/2020/07/Hydrogen-Line-Project-Documentation.pdf suggests 6.75cm.   As I understand it the 6.75cm is due to the probe being in a rectangular waveguide.

Should I move the probe to 6.75 or 6.5cm?

Thanks,
Nathan

**Comment by glangsto** [comment]
Hi Nathan,

Great hearing from you.   

The exact position of the feed probe actually depends on the shape of the horn you are building.
If you are building the horn with rectangular shape, then I’ll have to let someone else answer that
question, or maybe get you to do a test to determine the best answer yourself.

It would be great to hear more about your telescope.   Maybe send a picture?  
Also what 1st LNA are you using?  This will make the biggest difference in sensitivity of your telescope.

If you’ve seen the Milky Way, you’ve already done a great job.  The current probe must be 
close to the right place.  If you can, measure the system temperature
using the hot/cold load test described in LightWork Memo 4.   A more detailed test of just your 
amplifier alone is described in LightWork memo 28.

Memo 4: https://drive.google.com/file/d/1LSiBlAhl3gbjjDon6oFtzDrp0kMwBwkO/view?usp=sharing

Memo 28: https://drive.google.com/file/d/1Wgt05-DE5Kyz07wGalyl3w_PbZrketwF/view?usp=sharing

After you’ve made your first measurement, move the feed probe + 1 cm forward.
Measure the system temperature again.   Then move the feed probe back - 1 cm from the original position.
Measure the system temperature a 3rd time.

Now these three measurements will be on a parabola.   On a piece of paper draw a “parabolic” 
curve through the points.   The best feed probe position will be (hopefully) the point where the 
graph has the lowest system temperature.

This is a long answer to what might be a simple question for others. I’ve done a version of this
test and wrote up the results in memo 3.

Regards

Glen

Soon all LightWork memos will be in this shared directory:
https://drive.google.com/drive/folders/1SJJTUQ5Q6DLDuiqoSHR5YVY_0TDXaNIH?usp=sharing

One nice thing about google docs is that there is a good way for everyone to make suggestions in the
documents, then I can go in, check the suggestions and then incorporate them.

**Comment by glangsto** [comment]
Hi Nathan,

I now read more carefully your email.   So am guessing you built the rectangular feed.   

In re-reading the document you reference, they also describe the hot/cold load measurement (referencing
our earlier LightWork Memos).   

The authors of: http://physicsopenlab.org/wp-content/uploads/2020/07/Hydrogen-Line-Project-Documentation.pdf
did an excellent job of bringing together many references.

The system they built had excellent performance, with a
system temperature of about 100 K (although I could not completely understand their observing method).

There will be very little difference in the sensitivity of your telescope whether using 6.75 or 6.5 cm 
for the distance from the feed probe to the back of the can.   Again, by making two more measurements +/-1 cm from your
current position, you could make a very positive contribution to tuning the all telescopes for better performance.

Best regards

Glen

**Comment by nday91** [comment]
Glenn,

Thank you for responding.  I’ll do the measurements for the probe position you suggested and report back in a few days.

In the meantime, here is a description of my system and some pictures.

Antenna: DSPIRA Rectangular Feedhorn built using instructions from “Horn Construction” lesson.
LNA: Nooelec Sawbird H1 (attached to the probe SMA connector via a right angle adapter)
Receiver: Nooelec SmartSDR.
Processing: Raspberry Pi4 running Ubuntu.
Software: For the attached picture, specro_radiometer https://github.com/ccera-astro/spectro_radiometer


I’m tempted to spend another $20-$30 to add a Wideband LNA in between the H1 LNA and the SDR to boost the signal.

  
I did get the DSPIRA spectrum_w_cal.grc working also but the graph wasn’t as pretty - My calibration must’ve been off.

The shape of the “bump” in the graph changed as I scanned along the Milky Way and disappeared when I aimed away from the Milky Way.  So I’m confident I did observe the Hydrogen in the galactic plane as well as the Doppler shift.  Once I get the system optimized and my data collection sorted out I should able to re-create the velocity measurements.



Pictures attached:

**Comment by nday91** [comment]
The pictures didn't make in the previous post.

![Horn_end](https://user-images.githubusercontent.com/42848042/104662880-5b7b8380-5691-11eb-8e83-9623a77e89dd.jpg)
![Horn_side](https://user-images.githubusercontent.com/42848042/104662884-5ddddd80-5691-11eb-878c-67a393c0dbe0.jpg)
<img width="1135" alt="Hydrogen" src="https://user-images.githubusercontent.com/42848042/104662894-6209fb00-5691-11eb-95a4-a04fef5dc51a.png">

**Comment by GlenLangston** [comment]
Hi Nathan,

We’ve found that the GPIO Labs HI amplifier is about twice as sensitive as the
the NooElec HI version.  However the NooElec version has plenty of gain, so
adding another amplifier should not be necessary.

Note that in the excellent write-up you reference, they use the GPIO Labs LNA.

The LNA comparison measurements are documented in LightWork Memo 28:
https://drive.google.com/file/d/1Wgt05-DE5Kyz07wGalyl3w_PbZrketwF/view?usp=sharing

Your horn and the data appear good.   If you measure a hot-cold load equivalent temperature of 
under 200 Kevlin you are good to go.   Under 400 Kelvin is still OK, but observations just
take a little longer.    Above 400 Kelvin, then there is something about your horn
that you can readily fix, either the feed probe position or the first LNA quality/wiring
or some other aspect of the hardware.

Regards

Glen

**Reply by nday91** [reply]
Glen,

Thank you for the feedback on my system.

I attempted to recreate the distance from the end of the waveguide portion of your Lightwork Memo #3 and failed.   I drilled 3 more holes in my waveguide, 1 closer to the closed end and 2 farther from the closed end.  I moved the probe from hole to hole and measured the system temperature as described in memo 28.  I used the horn resting, pointed straight down at my concrete garage floor as the "hot" signal.  For the "cold" signal  I disconnected the horn between the Nooelec H1 LNA and the probe. 

I expected to find the maximum signal intensity and lowest system temperature at 5.25cm  from the end of the waveguide, as in memo #3 or perhaps at 6.75cm as others on the internet have recommended.  However, the signal increased as I moved the probe closer to the back of the waveguide even as close as 3.7cm.

Here is the system temperature data with respect to the distance between the probe and the closed end of the waveguide:
3.7cm: 148K
5.4cm: 220K
6.8cm: 243K
7.7cm: 266K

I think there is something fundamentally wrong with my process or procedure. Perhaps using the garage floor as a hot signal source is flawed, although I thought it would act as a broadband emitter from black body radiation.   Perhaps something else is changing as I move the probe closer to the end of the waveguide.  I believe a better test would be to measure the intensity of a generated 1.42GHz signal as In memo #3.  

However, I am giving up on antenna optimization for now as I think your work documented in memo #3 is very convincing.  I moved the probe back to the 5.25cm position so I can concentrate on gathering some data on the Milky Way. 

Thanks again for sharing your work. I will hopefully have some results to share soon.

Nathan

**Comment by glangsto** [comment]
Dear Nathan,

Thanks for your email.  

Your test setup and results are interesting.    I should probably run this test again
myself, but I do think you will have good results with observations of the sky.

One concern is that removing the LNA and feed probe, for the "cold load”, may remove the effect we are
trying to measure, the coupling of the feed probe to the horn shape.

Having the feed probe in the optimum location should change both the “hot” and “cold” 
load signals, making them both bigger in the optimum location.  So it is the
comparison of hot and cold load signals that gives the measure of efficiency.

(I think)

I’m looking forward to seeing your astronomical observations.

Regards

Glen

**Comment by GlenLangston** [comment]
Thanks to everyone for their help in our American Astronomical Society Demo.
That went very well I think.  

It was great that John and Joe had telescopes running during the presentation.
Joe’s use of VNC for sharing the telescope with students will be very useful.

Following up on the data sharing, the newest release of the Pi OS (Jan 20, 21)
automatically links all observations to the web page and allows world
wide viewing if VNC is enabled.

The latest release is 3.3 GBytes at 

https://drive.google.com/file/d/17nIbFazHjmZOEQav1sOTatBGRF_EYBuJ/view?usp=sharing

There are a few bug fixes, mostly to do with changing observing parameters. You
may not have noticed these bugs if you always observe in the same manner.

The installation guide is also available:
https://docs.google.com/document/d/1pStZsguO3UMrP0FhxJJJPFYdkg-Bp8TG3_zIh8MYHUQ/edit?usp=sharing

Have Fun,

Glen

You can distinguish the versions by the new background image.

**Comment by GlenLangston** [comment]
Hi Nathan,

How did your tests go?   It is critical to not change the telescope any between
the hot and cold loads.  No changes of amplifiers/gain settings or attenuation
for the tests to work.

Regards

Glen

**Reply by nday91** [reply]
Glen,

I cleaned up my configuration and had some success today.

My configuration is:
- Rectangular DSPIRA horn with 5.25cm probe, 5.25cm from closed end of wave-guide
- probe is soldered to a female SMA bulkhead connector
- SMA bulkhead connector is connected to a Nooelec Sawbird H1 LNa via a 90 degree female-female connector
- LNA connects to a Bias Tee using a 100cm wire
- Bias Tee is connected to a Nooelec Smart SDR via a SMA female-female connector
- Nooelec SDR is connected to Raspberry PI4 (Raspian Buster) via USB cable

I will attach some pictures of my setup and a screen capture showing the Hydrogen signal.  If anyone has feedback , I would appreciate comments.

I used NsfIntegrate24 to collect the data. When I select the system temperature tab while pointing at the Milky Way, the Median is 96.8 Kelvins.

Next I am working on a Arduino device to measure Elevation and Azimuth with a GPS fix and report back to the Raspberry PI via serial (USB or Bluetooth).  If I collect elevation, azimuth, longitude, latitude and time I should be able to have a good idea where the horn is pointing and translate to a variety of coordinate systems.  This might seem like overkill but I want to automate the data collection as much as possible before I start mapping the galaxy.

-Nathan

<img width="1337" alt="MW_67_350_counts" src="https://user-images.githubusercontent.com/42848042/106983378-6a52d480-672b-11eb-8557-cbdd96f6aa2e.png">

![Waveguid_Probe](https://user-images.githubusercontent.com/42848042/106983793-1c8a9c00-672c-11eb-88a0-8240439ce733.jpg)
![LNA_BiasTee](https://user-images.githubusercontent.com/42848042/106983893-3fb54b80-672c-11eb-8393-cf09762c1ecf.jpg)
![Horn](https://user-images.githubusercontent.com/42848042/106984021-73907100-672c-11eb-8188-ffc7f88c9053.jpg)

**Comment by glangsto** [comment]
Hi Nathan,

Your telescope seems to be working very well.  Great work!

You should have good results with mapping.

Note that the horn has about a 18 degree angular resolution, so you only need
pointing accuracy to about 5 degrees.   This is fairly easy to do with
a level and measurement of a few angles.

Your work with getting direct coordinate measures is great.   I’m also working on
a version of the Pi that uses the “Sense Hat” which has a compass, azimuth
and elevation (and roll pitch and yaw and acceleration) 

Attached is a picture of a compact telescope Pi, sense hat, GPS and SDRPLAY 
RSP1A (clone) in a clear plastic waterproof box.

I’ve not gotten all the angle measuring software working yet, but am hopeful
this device could directly measure the direction the horn is pointing.

Good Luck,

Glen

The SENSE hat has the very nice feature of also having a very bright 8x8 led
matrix, so that it comes with a method for scrolling information from
the Pi to the user, visible in daylight.

Everything for data taking and digital signal processing, except the very 
first LNA is in this plastic box.   It would be nice if the box was
a little bigger, but it all just fits.  The Pi computer is under the sense hat.
The GPS hat is visible on the top side of the Pi.



https://www.raspberrypi.org/products/sense-hat/ <https://www.raspberrypi.org/products/sense-hat/>

**Comment by glangsto** [comment]
Hi Nicole, Joe and Sue Ann,

I’ve finally fixed all the issues with the Pi OS on a bigger SD card.
The boundary between 32GB and 64GB caused some confusing issues.
The new operating system is being uploaded to Goggle Drive.  I’ll send a link
when it is ready.

I just sent, via priority US Mail, a feed probe and the 21-June-12 Pi OS on a
32GB SD card.  This was tested successfully on a Pi 3, a Pi 4B and
the Pi 400.    It was working with the AIRSPY (10 MHz), SDRPlay RSP1A (8 MHz)
and a NESDR SMArTee with bias tee (2.4 MHz).

Note that the more expensive SDRs have more gain and yielded better system
temperatures with the GPIOLabs HI LNA.   For the NESDR SMArTee SDR, the
NooElec HI amplifier has more gain so gave a better system temperature
with that SDR.   So if you’re using the cheapest SDR, probably you’d
prefer the slightly less expensive NooElec HI amplifier on Amazon.

Note that I’d recommend the AIRSPY or SDRPlay RSP1A if you can afford them.
They have much more gain than the cheaper SDRs.

For the SDRPlay RSP1A, the Chinese (Aliexpress version) is actually better functioning,
smaller and in a better metal container than the original. Plus comes with a USB
cable and a test antenna for playing with FM Radio.

Regards

Glen

Nicole your SD card will come soon; US Priority tracking #: 9114 9022 0078 9156 0729 31

Ali-express SDRPlay RSP1A screen capture:
***@***.***

Amazon Order.
***@***.***

Glen I Langston, Ph. D.
Galactic Astronomy Program Director
National Science Foundation

***@***.******@***.***>

On Jun 9, 2021, at 5:36 PM, Nicole Preiser ***@***.******@***.***>> wrote:

Hi Glen,

Sorry for the slow response. Grades and comments were due yesterday. For this reason I was also unable to attend the meeting. I wanted to attend but I just did not have the time.
I would love to have one of those microSD cards with the software loaded on it! We have all the parts, including the items you have in the picture. I think I only need the card with the
software so I can test it with the raspberry pi.

I've attached a photo of our telescope that you requested.

My address is 200 Country View, Arlington, VT 05250.

Take care,
Nicole

On Wed, Jun 9, 2021 at 3:14 PM Langston, Glen ***@***.******@***.***>> wrote:
Hi Nicole,

I’ve just received a few SD cards to write the Raspberry Pi operating system
for you.  I’ll send on one. Please email your USPS mail address.

If this works for you, please order from Amazon the replacement SD Crds and send
those to me.  I’ll send these out to the next teachers needing the Pi Os.

Here’s what I’ve ordered.

<Screen Shot 2021-06-09 at 3.06.17 PM.png>

I’ll also send you a feed probe, so you’ll have the beginning and end of the telescope,
but you will need to gather up the rest of the stuff!
<AficionadoFeedProbe+GPIOLabsHI-LNA.png>The picture shows an orange centimeter ruler, the feed probe, left under ruler,
and the GPIO Labs HI amplifier.   You’ll need to get your own amplifier.

Cheers

Glen


Glen I Langston, Ph. D.
Galactic Astronomy Program Director
National Science Foundation
2415 Eisenhower Avenue
Alexandria, VA 22314
P: 703-292-4937
C: 703-470-9820
***@***.***

On Jun 3, 2021, at 11:44 AM, Langston, Glen ***@***.***> wrote:

Hi Nichole,

We’ve got plenty of meetings, but could work in a
Zoom session with you.

Also I could send you an SD Card that has the PI OS
on it. Please send a picture of your horn telescope.

Best regards,

Glen

Glen I Langston, Ph. D.
Galactic Astronomy Program Director
National Science Foundation
304-456-3032
***@***.***

On Jun 2, 2021, at 8:58 PM, Nicole Preiser ***@***.***> wrote:

Hi Glen and Joe

It is a good thing I don't give up easily.
No, the suggestions didn't really make sense. I really have no experience with Raspberry Pi but if I read them again, I might be able to figure it out. I attempted to fix the SD card in a different way. I used an SD card formatter. The disk utility was saying there was no space even though there was nothing on the card. After reformatting it, I was able to copy the nsfsdr.image.sz to the micro SD but when I try to expand it, I get an error message (broken pipe). If I first expand it on my mac and then try to copy it, it says 'can’t be copied because it is too large for the volume’s format'.

It might be a good idea to zoom at some point soon.

I still have one more day of exams and then I will have some time. I will try to work on it more tomorrow.

Thanks for trying to help.

Take care,
Nicole


On Wed, Jun 2, 2021 at 11:38 AM Langston, Glen ***@***.***> wrote:
Hi Nichole,

Great hearing from you.

We recommend using a pi 4, but I think the pi 3 may work.

At what point do you get the “disk full” message?

On the pi, the latest "raspi-config" program can fix some of the issues with full
SD cards.

If the problem is on the pi, try these commands:

sudo raspi-config

In this program, update to the latest version of rasps-config

Then, once the latest version is installed, then run and go to
advanced options.

Then go to “expand filesystem”, this makes all the SD card space available.

Glen

PS.  It is probably better to get the radio-astronomy operating system
on a Mac or Windows machine and use Etcher to write the SDCard.   After
the SD card is written, use it on your raspberry pi.


Glen I Langston, Ph. D.
Galactic Astronomy Program Director
National Science Foundation
2415 Eisenhower Avenue
Alexandria, VA 22314
P: 703-292-4937
C: 703-470-9820
***@***.***

On Jun 2, 2021, at 10:48 AM, Nicole Preiser ***@***.***> wrote:

Hi Glen,

I would be interested to learn more about your summer program.

Ethan, my student, used this site to build the telescope https://wvurail.org/dspira-lessons/tour/

The telescope was finished last week. I am a little stuck with the raspberry pi. We are using a raspberry pi 3. It has a 16 GB micro SD card. I was not able to load the NsfSdr software (I get 'disk is full'). I purchased a 32 GB micro SD card and I have the same problem. After reading 'radio astronomy with Raspberry Pi' again, I think maybe I see the problem. I don't need to load the Raspberry Pi OS (from the Raspberry Pi website) and also the file provided in Google Drive (NsfSdr). Is this correct? I am going to try to load just the NsfSdr on the micro SD card and see if it works.

Take care,
Nicole



On Thu, May 27, 2021 at 10:04 PM Langston, Glen ***@***.***> wrote:
Hi Nicole,

It’s great you’re building and observing with your own telescope.

FYI we’re working on a summer program that might help you (and us).

The curriculum is still being developed but will be largely based
on the

WVURAIL.org/cra

documents

and the LightWork memos.

What are you using as your guide for building your telescope?

Cheers

Glen



Glen I Langston, Ph. D.
Galactic Astronomy Program Director
National Science Foundation
2415 Eisenhower Avenue
Alexandria, VA 22314
P: 703-292-4937
C: 703-470-9820
***@***.***

On May 27, 2021, at 1:10 PM, Nicole Preiser ***@***.***> wrote:

This email originated from outside of the National Science Foundation. Do not click links or open attachments unless you recognize the sender and know the content is safe.



Hi Glen and Joe,

Thanks for offering to chat with us. Today is the last day for seniors (Ethan is a senior) so it is a little hectic. We worked on trying to get the raspberry pi set up today. We don't have much experience so we still have some work to do. I will continue to work on it. If I can't figure it out, I will let you know.

Are the Tues/Thursday meetings at 2:30 pm ET or PT?

Take care,
Nicole

On Thu, May 27, 2021 at 10:49 AM Joe Wise ***@***.***> wrote:
I'm back from my MRI.

Let me know if you need to talk. Are you using Pi or PC? I forget.

You may have it working by now. 🙂

Joe


From: Langston, Glen ***@***.***>
Sent: Thursday, May 27, 2021 7:41 AM
To: Joe Wise ***@***.***>
Cc: Nicole Preiser ***@***.***>
Subject: Re: radio telescope


Hello Nichole,

I just read your email to Joe Wise.

I’m on Eastern Daylight Time.    (In West Virginia).

I could talk for a while today.  What time zone are you in?

Best regards

Glen


On May 27, 2021, at 1:24 AM, Joe Wise ***@***.***> wrote:

Nichole,

Sorry, but I am scheduled to have an MRI tomorrow morning at that time.

Perhaps Glen Langston has some time.

I have cc'd him here.

I'll check in with you tomorrow. Good luck.

Joe

From: Nicole Preiser ***@***.***>
Sent: Wednesday, May 26, 2021 6:02 PM
To: Joe Wise ***@***.***>
Subject: Re: radio telescope

Hi Joe,

Ethan, my student, has completed the build and we are going to try to get the electronics working tomorrow. The last bit we needed arrived today.
Would you be available to chat (zoom) tomorrow at 10 am ET (7 am your time)? This is very last minute, so I totally understand if this is not possible for you.
Also, I am not sure if you would need to be leaving for school or if you are even teaching in person at the moment.
If this does not work for you, I can try to see if I can try to see if I can get the telescope to work and then send questions or we can try to meet at a different time.
I will try to attend the Tues/Thursday 2:30 meetings soon. Are the meetings 2:30 pm your time or my time or something else?

Take care,
Nicole


Glen I Langston, Ph. D.
Galactic Astronomy Program Director
National Science Foundation
Telework: 304-456-3032
***@***.***





<IMG_3160.jpg>

**Comment by glangsto** [comment]
Hi Glen,

Here are some photos of the electronics (the airspy R2 SDR, the GPIO labs
LNA) that we have. The tech department at school also said they would order
a raspberry pi 4 for me. Once I have the software on the micro SD card, I
think I should be able to test the telescope. I did purchase a 32 GB card
but I had some trouble loading the software (as I explained earlier). Once
the card arrives from you, I think I'll have everything I need.
(I attempted to send this note but it said 'message too large' so I will
try to send the photos in sets of two.

**Comment by glangsto** [comment]
Dear Nathan,

Thanks for this email.  I found that my response to your excellent work was never sent.

You’ve built a telescope with excellent system temperature.   Hope you’ve been able to 
make some good observations.

Best regards

Glen

**Comment by glangsto** [comment]
Hello DSPIRA and all,

At our last DSPIRA meeting on Thursday October 21 we
talked about Jupyter notebooks for looking at data.

I’d been using the NOIRALab.edu’s datalab, which provides free accounts
for looking at any astronomical observations.

John Makous noted that he already uses “Google Collaboratory” with his students,
and likes the simplicity of it.

It turned out that my notebook needed a couple adjustments to work
inside the collaboratory.

To look at some horn radio telescope data I think you only need to 
either copy this file to your computer or tell the Google Chrome notebook to
look at this file:

https://github.com/glangsto/jupyter/blob/master/MilkyWayPlottingColab.ipynb

Then notebook downloads all the data (somewhere) (not too much data), and
the if you step through the notebook it creates plots of the Milky Way.

Please give this a try and see if it works for you.

Take care

Glen

**Comment by GlenLangston** [comment]
Hello

This is an update to my previous email.  
After saving the linked file, to run it, do the following.

1) Open Google Chrome
2) Within Chrome open your Google Drive
3) In the Google Drive, pull down “New” until you see
4) Google Collaboratory
5) The collaboratory will start, then under “File” 
6) Run “Upload Notebook” and then go to the place you saved the notebook
7) The notebook will start running.
8) Simultaneously hold down the SHIFT and RETURN keys to execute the notebook.

Good luck!

Glen

> On Oct 25, 2021, at 4:06 PM, Glen Langston ***@***.***> wrote:

Save this file 

> https://github.com/glangsto/jupyter/blob/master/MilkyWayPlottingColab.ipynb

**Comment by glangsto** [comment]
Hello DSPIRA and all,

At our last DSPIRA meeting on Thursday October 21 we
talked about Jupyter notebooks for looking at data.

I’d been using the NOIRALab.edu’s datalab, which provides free accounts
for looking at any astronomical observations.

John Makous noted that he already uses “Google Collaboratory” with his students,
and likes the simplicity of it.

It turned out that my notebook needed a couple adjustments to work
inside the collaboratory.

To look at some horn radio telescope data I think you only need to 
either copy this file to your computer or tell the Google Chrome notebook to
look at this file:

https://github.com/glangsto/jupyter/blob/master/MilkyWayPlottingColab.ipynb

Then notebook downloads all the data (somewhere) (not too much data), and
the if you step through the notebook it creates plots of the Milky Way.

Please give this a try and see if it works for you.

Take care

Glen

We’re meeting using Joe Wise’s Zoom link (below)
That is 4:00 pm Eastern time.

https://wildwood.zoom.us/j/2799196673 <https://wildwood.zoom.us/j/2799196673>
>

**Comment by EvrettSob** [comment]
Hello!

I was wondering, as for the insulated lining that comprises the 'dish' of the telescope, is there a recommended thickness of this material? My professor estimates we have a material that's possibly an inch thick, give or take. Does this matter, and do you have any suggestions? Thank you!
